packages/ChangesReloaded-Core.package/ChangeRecord.extension/instance/isInImage.st

```diff
@@ -0,0 +1,19 @@
+*ChangesReloaded-Core
+isInImage
+	
+	self type = #method
+		ifTrue: [| class selector |	
+			class := self methodClass.
+			selector := self methodSelector.
+			class ifNil: [^ false].
+			(class lookupSelector: selector) ifNil: [^ false].
+			"TODO: Move this method to TimeStamp (TimeStamp >> fromStamp: ) --> strips away author initials from actual timestamp"
+			^ (TimeStamp fromAuthorStamp: (class lookupSelector: selector) timeStamp) >= self timeStamp].
```

Comment:
```suggestion
		ifTrue: [| class selector method |	
			class := self methodClass.
			selector := self methodSelector.
			method := class lookupSelector: selector.
			class ifNil: [^ false].
			method ifNil: [^ false].
			"TODO: Move this method to TimeStamp (TimeStamp >> fromStamp: ) --> strips away author initials from actual timestamp"
			^ (TimeStamp fromAuthorStamp: method timeStamp) >= self timeStamp].
```
I think it is more readable like this, especially in the last line.
(Does `method := class lookupSelector: selector.` actually work like this or are brackets needed?)

Reply:
Will implement the idea, the current suggestion has a bug though (`class lookupSelector: selector` is called before checking if class is Nil)